sdk/src/trace/span.cc

```diff
@@ -55,6 +55,12 @@ Span::Span(std::shared_ptr<Tracer> &&tracer,
   processor_->OnStart(*recordable_);
   recordable_->SetName(name);
 
+  for (auto &attr : options.attributes)
+  {
+    auto value = attr.second;
```

Comment:
Is this copying the attribute value intentionally?

Reply:
No. Fixed.